angularjs-portal-home/src/main/webapp/my-app/search/partials/search-results.html

```diff
@@ -5,23 +5,28 @@
         <a>All ({{ totalCount }})</a>
       </li>
       <li ng-click="filterTo('myuw')" id="myuw-selector">
-        <a>MyUW ({{myuwFilteredResults.length}})</a>
+        <a>MyUW <span ng-hide="portletListLoading">({{myuwFilteredResults.length}})</span>
+        <i ng-show="portletListLoading" class="fa fa-spinner fa-pulse"></i></a>
```

Comment:
Additionally we should probably be consistent with the flags we are using for loading state. Currently we are checking both the length, and these flags.